src/FE/FEsrc/pages/ProfileCreation.tsx

```diff
@@ -208,11 +208,11 @@ const ProfileCreationPage = (props) => {
                     snackText={"Account Registered Successfully!"}
                 />
                 <FormContainer>
-                    {!props.editing && isClinician && <FormTitle>New Patient Profile</FormTitle>}
-                    {!props.editing && !isClinician && <FormTitle>New Search Profile</FormTitle>}
-                    {props.editing && isClinician && <FormTitle>Edit Patient Profile</FormTitle>}
-                    {props.editing && !isClinician && <FormTitle>Edit Search Profile</FormTitle>}
-                    {!props.editing && isClinician && <FormDisclaimerText>
+                    {!props.editing && (localStorage.getItem('isClinician') == "true") && <FormTitle>New Patient Profile</FormTitle>}
+                    {!props.editing && !(localStorage.getItem('isClinician') == "true") && <FormTitle>New Search Profile</FormTitle>}
+                    {props.editing && (localStorage.getItem('isClinician') == "true") && <FormTitle>Edit Patient Profile</FormTitle>}
+                    {props.editing && !(localStorage.getItem('isClinician') == "true") && <FormTitle>Edit Search Profile</FormTitle>}
+                    {!props.editing && (localStorage.getItem('isClinician') == "true") && <FormDisclaimerText>
```

Comment:
Needed to add this - otherwise, upon registering, it was for a brief second showing the patient view, i.e., "New Search Profile", then switching to the clinician view.